src/plugins/intel_cpu/src/emitters/snippets/x64/jit_brgemm_copy_b_emitter.cpp

```diff
@@ -81,31 +82,41 @@ void jit_brgemm_copy_b_emitter::emit_impl(const std::vector<size_t>& in, const s
     if (out.size() > 1)
         mem_ptrs_idxs.emplace_back(out[1]);
 
+    std::set<snippets::Reg> regs_to_spill = m_live_regs;
+    // Note: these 3 registers will be corrupted by the caller during the ABI call
+    regs_to_spill.emplace(snippets::RegType::gpr, abi_param1.getIdx());
+    regs_to_spill.emplace(snippets::RegType::gpr, abi_param2.getIdx());
+    regs_to_spill.emplace(snippets::RegType::gpr, h->rbp.getIdx());
+    // Note: abi_param_1 is a default invalid value to check later that the aux reg was allocated properly
+    Xbyak::Reg64 aux_reg = abi_param1;
+    const bool is_dynamic_case =
+        std::any_of(m_memory_offsets.cbegin(), m_memory_offsets.cend(), ov::snippets::utils::is_dynamic_value<size_t>);
+    if (is_dynamic_case)
+        aux_reg = utils::init_memory_access_aux_gpr(mem_ptrs_idxs, aux_gpr_idxs, regs_to_spill);
+
     EmitABIRegSpills spill(h);
-    spill.preamble();
+    spill.preamble(regs_to_spill);
```

Comment:
There are some places where `preamble` is called without arguments: perf counters emitters, `TPPEmitter`. Do I understand correctly that ideally we need to use there the same logic as we have in x64 brgemm-specific emitters?

Reply:
Ideally, but not necessarily. After all, spilling live registers is not a requirement, but an optimization.
Keeping that in mind, we can use the same approach, but it's not strictly necessary. Although it will likely significantly improve TPP perf for elwise subgraphs.

Reply:
Got it, thanks!